Gemfile

```diff
@@ -5,13 +5,12 @@ source 'https://rubygems.org'
 
 ruby '>=2.3.1'
 
-#gem 'rails', '5.2.4.4'
-gem 'rails', '5.2.6'
+gem 'rails', '~> 6.0'
```

Comment:
By definition (ruby versioning policy) this should be safe, but I'm not exactly sure if having the rails itself autoupdate is what we want to do.

Reply:
Yes. I think we want to get patch updates, though, so '~> 6.1.6' should be good?